shared/ios/Keybase/Pusher.m

```diff
@@ -33,8 +33,12 @@ - (void)localNotification:(NSString*)ident msg:(NSString*)msg badgeCount:(long)b
 
 - (void)displayChatNotification:(KeybaseChatNotification *)notification {
   NSString* ident = [NSString stringWithFormat:@"%@:%ld", notification.convID, notification.message.id_];
-  NSString* msgTxt = notification.isPlaintext ? notification.message.plaintext : @"Encrypted Message...";
-  NSString* msg = [NSString stringWithFormat:@"%@ (%@): %@", notification.message.from.keybaseUsername, notification.conversationName, msgTxt];
+  NSString* msg;
+  if (notification.isPlaintext && [notification.message.plaintext length] != 0) {
+    msg = [NSString stringWithFormat:@"%@ (%@): %@", notification.message.from.keybaseUsername, notification.conversationName, notification.message.plaintext];
```

Comment:
what do you think of this instead:
```
   NSString* ident = [NSString stringWithFormat:@"%@:%ld", notification.convID, notification.message.id_];
   NSString* msg;
   if (notification.isPlaintext && [notification.message.plaintext length] != 0) {
-    msg = [NSString stringWithFormat:@"%@ (%@): %@", notification.message.from.keybaseUsername, notification.conversationName, notification.message.plaintext];
+      if([ notification.message.from.keybaseUsername isEqualToString: notification.conversationName ]) {
+        msg = [NSString stringWithFormat:@"%@ sent you a message: %@", notification.message.from.keybaseUsername, notification.message.plaintext];
+      } else {
+        msg = [NSString stringWithFormat:@"%@ sent you a message in %@: %@", notification.message.from.keybaseUsername, notification.conversationName, notification.message.plaintext];
+      }
   } else {
     msg = notification.message.serverMessage;
   }
```

Reply:
That plaintext non 1-1 case is too verbose no?

Reply:
i preferred it but up to you, we can always tweak these. i think it's a slightly better UX with the verbose verison